src/devices/Ft4222/Ft4222.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+ <ItemGroup>
+    <Compile Include="*.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
+      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=win</AdditionalProperties>
```

Comment:
@joperezr how should we correctly add also linux in the next PR? Package reference instead for this specific project?

Reply:
Here we should also use Linux as the RuntimeIdentifier. This is not so the right asset gets picked at runtime, the only reason we pass in linux is in order to use the linux reference assembly when compiling this sensor, and it is important to always use the Linux version of the implementation when compiling as that one is the true reference assembly. For trying this code out locally, you can change it to win if you want or you can add a package reference instead of this which would also work, but for the version we check in please make sure you set the runtime identifier to be Linux

Reply:
OK, I'll change back to Linux. I'm currently working on having a proper Linux (and MacOS) version as well. so when I4ll reach this point, what should I pick to check in the code?

Reply:
You should always pick Linux. In the end, once we actually publish the Iot.Device.Bindings package, it will use the right one depending on where you publish for, but for development time we need always the Linux one as that one is the correct reference assembly.